cmd/tools/grafana/dashboard_test.go

```diff
@@ -699,6 +699,8 @@ func checkVariablesHaveAll(t *testing.T, path string, data []byte) {
 	}
 	exceptionForAllValues := map[string]bool{
 		"cmode/security.json": true,
+		"cmode/cluster.json":  true,
```

Comment:
still needed?

Reply:
yes it's needed, As we removed `.*` from the `Cluster` var.